snaps_boot/provision/rebar_utils.py

```diff
@@ -109,9 +109,11 @@ def __create_images():
     """
     # TODO/FIXME - find appropriate API to perform these tasks
     logger.info('Setting up Digital Rebar images')
+    pack_dir = pkg_resources.resource_filename('snaps_boot', 'drp_content')
     playbook_path = pkg_resources.resource_filename(
         'snaps_boot.ansible_p.setup', 'drp_images_create.yaml')
-    ansible_utils.apply_playbook(playbook_path)
+    ansible_utils.apply_playbook(playbook_path, variables={
```

Comment:
Why has this variable been added? It does not look like the playbook has anything to substitute...

Reply:
Good catch. Updated to be the same as trunk.